src/Plugin/resource/Decorators/ResourceDecoratorBase.php

```diff
@@ -309,4 +309,26 @@ public function getUrl(array $options = array(), $keep_query = TRUE, RequestInte
     return $this->subject->getUrl($options, $keep_query, $request);
   }
 
+  /**
+   * If any method not declared, then defer it to the decorated field.
+   *
+   * This decorator class is proxying all the calls declared in the
+   * ResourceInterface to the underlying decorated resource. But it is not
+   * doing it for any of the methods of the parents of ResourceInterface.
+   *
+   * With this code, any method that is not declared in the class will try to
```

Comment:
If the method does not exist we are falling back to PHP's native behavior, which is a fatal error.

Something like
```php
$decorator->validMethodInDecorator();
```
and
```php
$decorator->validMethodInDecorated();
```
will yield expected results.

Something like:

```php
$decorator->invalidMethod()
```
will also result in the expected behavior, which is a fatal error.